.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "env": {
+    "es6": true,
+    "node": true
+  },
+  "extends": "airbnb"
+}
```

Comment:
We should convert this to `.eslintrc.yml`... can address later, but may as well make that change when adding the newline.